tests/io/lammps/test_data.py

```diff
@@ -562,18 +562,18 @@ def test_init(self):
         # q and v from site properties, while type from species_string
         topo = Topology(sites=mol)
         assert topo.type_by_sites == ["H"] * 10
-        np.testing.assert_array_equal(topo.charges, inner_charge)
-        np.testing.assert_array_equal(topo.velocities, inner_velo)
+        assert_array_equal(topo.charges, inner_charge)
+        assert_array_equal(topo.velocities, inner_velo)
         # q and v from overriding, while type from site property
         topo_override = Topology(sites=mol, ff_label="ff_map", charges=outer_charge, velocities=outer_velo)
         assert topo_override.type_by_sites == ["D"] * 10
-        np.testing.assert_array_equal(topo_override.charges, outer_charge)
-        np.testing.assert_array_equal(topo_override.velocities, outer_velo)
+        assert_array_equal(topo_override.charges, outer_charge)
+        assert_array_equal(topo_override.velocities, outer_velo)
         # test using a list of sites instead of SiteCollection
         topo_from_list = Topology(sites=mol.sites)
         assert topo_from_list.type_by_sites == topo.type_by_sites
-        np.testing.assert_array_equal(topo_from_list.charges, topo.charges)
```

Comment:
`assert_array_equal` usage across `io.lammps.test_data` is very likely incorrect, looks like float array